WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -111,6 +112,9 @@ protected WalletViewModel(Wallet wallet) : base(wallet)
 		this.WhenAnyValue(x => x.HeightSource)
 			.Subscribe(x => LayoutSelector(_widthSource, x));
 
+		this.WhenAnyValue(x => x.IsWalletBalanceZero)
+			.Subscribe(_ => IsSendButtonVisible = !IsWalletBalanceZero && (!wallet.KeyManager.IsWatchOnly || wallet.KeyManager.IsHardwareWallet));
+
```

Comment:
I might be wrong but all the Hardware wallets are also WatchOnly wallets as well.

So one check only for the WatchOnly would be enough?